scripts/gas-refund-program/pip38/index.ts

```diff
@@ -0,0 +1,96 @@
+import { assert } from 'ts-essentials';
+import fantomEpoch36 from './merkletree-chain-250-epoch-36.json';
+import fantomEpoch37 from './merkletree-chain-250-epoch-37.json';
```

Comment:
```suggestion
import * as fantomEpoch36 from './merkletree-chain-250-epoch-36.json';
import * as fantomEpoch37 from './merkletree-chain-250-epoch-37.json';
```

didn't work out for me until I did this

did it for you?